### PR TITLE
Added API to check address type and convert to json

### PR DESCRIPTION
**Pull request by thangnc**
- Added API to check MERGE fields type
- Convert string to JSON if MERGE field's type is `address`
- Removed some private redundant methods

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 0.3.5.2 - 2017-06-06
+- Removed log query and add API to check address type [#22](https://github.com/treasure-data/embulk-output-mailchimp/pull/22)
+
```
Comment (sakama) [inline CHANGELOG.md]: I think it's better to just add tag when you want to release minor version.
Then, include following changes when you release public release version.
* Update CHANGELOG
* Update build.gradle

**Review by sakama — COMMENTED** [review]
Put 1 comment.
Other parts looks good to me.

Please remind that this plugin is OSS based plugin.
We should keep our changes clear to any developers and users.

File: CHANGELOG.md (hunk 1 above)
Comment (thangnc) [reply CHANGELOG.md]: Thanks @sakama. I will move changes in 0.3.6 versions

**Comment by thangnc** [comment]
@sakama i removed change log of minor versions. Will move them back when release major version 0.3.6

**Review by sakama — APPROVED** [review]
minor version still exists at build.gradle.
Please read my above comments again.
Other parts looks good to me.

**Comment by thangnc** [comment]
@sakama I downgraded version in build.gradle

**Pull request merged**